src/microbe_stage/biomes.cpp

```diff
@@ -99,6 +102,16 @@ Biome::Biome(Json::Value value)
         std::vector<std::string> compoundChunkNames =
             chunkCompoundData.getMemberNames();
 
+        // Calculate iron values to send to GUI
+        if(name == "Small Iron Chunk") {
+            smallIronChunk =
+                chunkCompoundData["iron"]["amount"].asDouble() * density;
+
+        } else if(name == "Big Iron Chunk") {
+            bigIronChunk =
+                chunkCompoundData["iron"]["amount"].asDouble() * density;
+        }
```

Comment:
This data already goes to the chunk data. Instead of hardcoding 2 chunk types like this, I think the chunk compound amounts should be extracted from the  `chunks` member variable.
https://github.com/Revolutionary-Games/Thrive/commit/9990b42544700792b1d56b0108b025821ef76d6c#commitcomment-35578503